### DISCUSSION TITLE
How to use custom stylesheets for dark mode and other stuff

### DISCUSSION THREAD

**Discussion by pranshu-raj-211** [google/mesop#446] [Q&A]
I'm making a couple of websites using mesop, and I'd like to implement a dark mode toggle, cards among other stuff. Could you point me to some example in this repo where something similar has been done, so that I can make it and submit a PR?

Thanks for the wonderful work that you're doing!

**Comment by wwwillchen** [comment]
I opened an issue #461 - we don't have any examples of dark theme since we don't support it yet.

**Comment by lnarimatsu** [comment]
Interesting, I'm having trouble adapting the colors. I can change the color of a button, for example, but not the borders of an input field when it is focused.

It would be nice to have an easier way to change the default color tokens for primary, secondary, etc.
